src/main/java/org/springframework/cloud/scheduler/spi/cloudfoundry/CloudFoundryAppScheduler.java

```diff
@@ -118,11 +167,21 @@ private void scheduleJob(String appName, String scheduleName, String expression,
 					build());
 		})
 				.onErrorMap(e -> {
-					throw new CreateScheduleException("Failed to schedule: " + scheduleName, e);
+					if (e instanceof SSLException) {
+						throw new CloudFoundryScheduleSSLException("Failed to schedule" + jobName, e);
+					}
+					else {
+						throw new CreateScheduleException("Failed to schedule: " + jobName, e);
+					}
```

Comment:
Why are we throwing the CF specific exception here?  Why is wrapping the exception not good enough?

Reply:
This is an Exception within the CloudFoundAppScheduler project.   Should I rename it?